pkg/nftables/networkpolicy.go

```diff
@@ -125,9 +127,12 @@ func clusterwideNetworkPolicyEgressToRules(e firewallv1.EgressRule) (allow, exce
 
 func clusterwideNetworkPolicyEgressToFQDNRules(
 	cache FQDNCache,
+	fqdnState firewallv1.FQDNState,
 	e firewallv1.EgressRule,
 ) (rules []ruleBase, updatedState firewallv1.FQDNState) {
-	fqdnState := firewallv1.FQDNState{}
+	if fqdnState == nil {
+		fqdnState = firewallv1.FQDNState{}
+	}
```

Comment:
What is this change for? This function is used only once and an empty map is passed.